README.md

```diff
@@ -165,10 +165,11 @@ While some implicit technical details are obvious to you, they might not be for
 
 ## Code examples
 
+* Open and close code blocks with ````shell` where shell is the language for syntax highlighting
 * Try to keep code examples together in one block, avoid mixing paragraphs and code blocks
 * Code blocks follow standard markdown formatting
 * Show output only when necessary
-* Output should be a comment
+* Output should be a comment `#`
```

Comment:
I don't agree with this either, because that's not how it appears in the terminal.